### ISSUE TITLE
Update to 3.1.0/3.1.0b from 3.0.4 sends lights objects spawn by the platform switch into unavailable state

### ISSUE THREAD

**Issue by gribouk** [nielsfaber/scheduler-card#375]
HA core 2021.02.3, supervised

It has been noticed, that the last 2 updates send the lights, which were derrived from the platform switch into unavailable state after the first reboot uppon update. To be specific, let us have template switch:
```
switch:
  - platform: template
    switches:
      hall_shandalier:
          value_template: |-
              {{
                  is_state('switch.hall_shandalier_down', 'on') 
                  and 
                  is_state('switch.hall_shandalier_switch', 'on') 
                  and
                  is_state('switch.hall_shandalier_top', 'on') 
              }}
          turn_on:
            service: switch.turn_on
            data:
              entity_id: switch.hall_shandalier_down, switch.hall_shandalier_switch, switch.hall_shandalier_top
          turn_off:
            service: switch.turn_off
            data:
              entity_id: switch.hall_shandalier_down, switch.hall_shandalier_switch, switch.hall_shandalier_top
```
which is destened to synchronise 3 physical switches near 3 different exits - works fine for 4+ month.

Then we would want to convert it into the lights domain for the obvious benefit reasons, using standard platform switch for this purpose:

```
  - platform: switch
    name: hall_shandalier
    entity_id: switch.hall_shandalier
```

works fine as well for quite a long time.

Now do the update for scheduler component and reboot - it will never turn available anymore... Double checked several times - it is a sheduler integration that halts this thing.

**Comment by KTibow** [comment]
Anything in logs?

**Comment by nielsfaber** [comment]
I don't understand this issue.
If an entity becomes unavailable, this is out of control of scheduler.
I do not know in which conditions a template switch becomes unavailable.

**Comment by gribouk** [comment]
Neither do I. But the fact is that under the HA core 2021.02.3, once you update to 3.1.0 and reboot, the lights from switch platform simply no longer available (do not load anymore).

I decided to check if it is the case with the newer versions and updated HA to 2021.4.6, and discovered that there is no such problem as experienced under 2021.02.3.

The reason I have to use HA 2021.02.3 up to this day is in the script editor bug introduced with March release, which makes one impossible to use it in the yaml mode. I was promissed that by 2021.5 it would be solved, untill then I just do not update to 3.1.0.
